packages/configs/babel/babel.config.js

```diff
@@ -2,6 +2,16 @@ const BABEL_ENV = process.env.BABEL_ENV;
 
 const defaultPlugins = [['babel-plugin-styled-components', { pure: true }]];
 
+const browsers = [
+  'last 2 Chrome major versions',
+  'last 2 Firefox major versions',
+  'last 2 Safari major versions',
+  'last 2 Edge major versions',
+  'last 2 Android major versions',
+  'last 2 ChromeAndroid major versions',
+  'last 2 iOS major versions',
```

Comment:
Curious on why you chose these?

Wondering if we can just use `defaults` instead (`> 0.5%, last 2 versions, Firefox ESR, not dead`)?

Reply:
We are currently using `defaults`. We don't officially support all of them:

```
and_chr 91
and_ff 89
and_qq 10.4
and_uc 12.12
android 91
baidu 7.12
chrome 91
chrome 90
chrome 89
edge 91
edge 90
firefox 89
firefox 88
firefox 78
ie 11
ios_saf 14.5-14.6
ios_saf 14.0-14.4
ios_saf 13.4-13.7
kaios 2.5
op_mini all
op_mob 62
opera 76
opera 75
safari 14.1
safari 14
samsung 14.0
samsung 13.0
```